components/menu/index.css

```diff
@@ -11,849 +11,945 @@ governing permissions and limitations under the License.
-        + var(--spectrum-menu-item-text-to-control)
-        + var(--spectrum-menu-item-icon-width)
-        + var(--spectrum-menu-item-label-text-to-visual)
-        + var(--spectrum-menu-item-focus-indicator-width)
-    ));
-
-  /* "no" icon: just the chevron (we're not counting it because it HAS to be there for a collapsible) */
-  --spectrum-menu-item-collapsible-no-icon-submenu-item-padding-x-start:
-    calc((
-      var(--spectrum-menu-item-label-inline-edge-to-content)
-        + var(--spectrum-menu-item-checkmark-width)
-        + var(--spectrum-menu-item-label-text-to-visual)
-        + var(--spectrum-menu-item-focus-indicator-width)
-    ));
+	--spectrum-menu-item-background-color-default-rgba: rgba(var(--spectrum-menu-item-background-color-default), var(--spectrum-menu-item-background-opacity));
+	--spectrum-menu-item-background-color-key-focus: var(--spectrum-menu-item-background-color-keyboard-focus);
+	--spectrum-menu-item-min-height: var(--spectrum-component-height-100);
+	--spectrum-menu-item-corner-rounding: var(--spectrum-corner-radius-500);
+	--spectrum-menu-item-icon-height: var(--spectrum-workflow-icon-size-100);
+	--spectrum-menu-item-icon-width: var(--spectrum-workflow-icon-size-100);
+	--spectrum-menu-item-thumbnail-height: var(--spectrum-thumbnail-size-700);
+	--spectrum-menu-item-thumbnail-width: var(--spectrum-thumbnail-size-700);
+	--spectrum-menu-item-label-font-size: var(--spectrum-font-size-100);
+	--spectrum-menu-item-label-text-to-visual: var(--spectrum-text-to-visual-100);
+
+	--spectrum-menu-item-label-inline-edge-to-content: var(--spectrum-component-edge-to-text-100);
+	--spectrum-menu-item-top-edge-to-text: var(--spectrum-component-top-to-text-100);
+	--spectrum-menu-item-bottom-edge-to-text: var(--spectrum-component-bottom-to-text-100);
+	--spectrum-menu-item-top-to-icon: var(--spectrum-component-top-to-workflow-icon-100);
+	--spectrum-menu-item-top-to-drillin: var(--spectrum-menu-item-top-to-disclosure-icon-medium);
+	--spectrum-menu-item-top-to-thumbnail: var(--spectrum-menu-item-top-to-thumbnail-medium);
+	--spectrum-menu-item-thumbnail-to-label: var(--spectrum-text-to-visual-200);
+
+	--spectrum-menu-item-text-to-control: var(--spectrum-text-to-control-100);
+
+	--spectrum-menu-item-description-font-size: var(--spectrum-font-size-75);
+
+	--spectrum-menu-section-header-font-size: var(--spectrum-font-size-100);
+	--spectrum-menu-section-header-min-width: var(--spectrum-component-height-100);
+	--spectrum-menu-section-description-font-size: var(--spectrum-font-size-75);
+	--spectrum-menu-section-header-to-description: var(--spectrum-menu-section-header-to-description-medium);
+
+	--spectrum-menu-item-selectable-edge-to-text-not-selected: var(--spectrum-menu-item-selectable-edge-to-text-not-selected-medium);
+
+	--spectrum-menu-item-checkmark-height: var(--spectrum-checkmark-icon-size-100);
+	--spectrum-menu-item-checkmark-width: var(--spectrum-checkmark-icon-size-100);
```

Comment:
These are pointing to the incorrect values, right? It looks like 75 & 100 are both 10px right now, but that's not how it should be according to Figma, correct?

Reply:
Correct, it looks like the tokens are not pointing to the same values in Figma, but they're pointing to the right tokens--something to call out when this gets design reviewed. Thanks!